src/lib/parseASN.ts

```diff
@@ -0,0 +1,22 @@
+import type { ParsedASN } from "src/types/ParsedASN"
```

Comment:
FYI: there's already some code for parsing an ASN - see `lib/asn.ts`. Though to be fair, that code doesn't handle 21 char ASNs